### DISCUSSION TITLE
Connect new orchard core instance to existing db with data

### DISCUSSION THREAD

**Discussion by TimAht1** [OrchardCMS/OrchardCore#11122] [Q&A]
I am trying to figure out how to connect a new orchard core instance to existing db tables with data. For example: the existing orchard core instance were accidentally deleted. Now we need to install a new instance and connect it to existing data in db. As I understand, setup page is trying to create same tables again and not trying to connect to existing data. For db we use AzureSql.

**Comment by hishamco** [comment]
I think we may need an option to avoid setup to create the required table. Did you tried an `AutoSetup` module, but I need to make sure if it does the same thing without setup screen or it has flixibility to connect to the database without creating tables

I will wait for someone to confirm

**Comment by Skrypt** [comment]
You may need to have the DataProtection tokens generated by the previous installation. Else, it will fail miserably. Then, the only thing to do is to have the App_Data folder and a tenants.json file with a tenant configuration that has a proper running state.

**Reply by hyzx86** [reply]
Hi @Skrypt , I deleted my  data protection tokens  ... Can it recover?
https://github.com/OrchardCMS/OrchardCore/discussions/13945
